### ISSUE TITLE
Buddy Press profile select filter doesn't work

### ISSUE THREAD

**Issue by eri-trabiccolo** [Nikeo/customizr#1606]
This seems to be due to the fact that the Buddy Press select element wrapper has the too much generic `filter` CSS class, the same too much generic CSS class we use to apply the slider overlays:
https://github.com/presscustomizr/customizr/blob/dev/assets/front/css/style.css#L2966

The solution would be to "override" that for the buddypress filter, or, better, prefix our `filter` CSS class, to avoid compatibility problems in the future.

Basically that "before" overlaps the select so that it becomes not "not clickable".
Other minor CSS improvements can be done to that select appearance.

see: https://github.com/presscustomizr/customizr/issues/1532#issuecomment-430184314
